doc-content/enterprise-only/openshift/environment-managed-con.adoc

```diff
@@ -8,10 +8,6 @@ Smart Router is a single endpoint that can receive calls from client application
 
 You must provide a Maven repository with the processes (KJAR files) that you want to deploy on the servers. Your integration process must ensure that the required versions of the processes are uploaded to the Maven repository. You can use {CENTRAL} in a development environment to create the processes and upload them to the Maven repository.
 
-{PRODUCT} includes two base templates for managed environments:
+{PRODUCT} includes the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` base templates a for managed environment. It includes two {KIE_SERVER} pods (with database pods), Smart Router in a high-availability configuration, and {CENTRAL} monitoring in a high-availability configuration.
```

Comment:
So you have one file, that contains more than one template. So I'd probably say ...yaml file that includes templates.. typo a for
Unclear what it is? This YAML file includes...